### PR TITLE
Adding docs for iframe integration

### PR DESCRIPTION
**Pull request by yowlu**
This PR implements adding docs for the upcoming iframe feature so merchants will be able to understand what they need to do in order to add our iframe in their site.

Please read it and make sure it is understandable enough for a merchant to start using our iframe. Otherwise, propose changes or let me know what it is not clear.

### CODE REVIEW

**Review by jproyo — APPROVED** [review]
Great doc

File: source/includes/_iframe.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	- `min-height: 560px`: Although this one it is just recommended, having this minimum height will make scrollbars disappear.
+
+
+## Communicating with the iframe events
+
+The iframe contains both part of the implementation the merchant would have on their side as well as some implementation that usually happens on the wallet side.
+
+Merchant would usually need to gather from the users what fiat amount and fiat currency they want to deposit and send it to the ON/RAMP api using the `create_ingress_invoice` endpoint to later redirect the users to the redirection url they got as a response from ON/RAMP. However, in this integration, the iframe will deal with gathering the information from the user and redirecting the user to the ON/RAMP wallet only if the payment was successful.
+
+Given that, merchant and the iframe need a way to communicate certain events and this is achieve by using the `window.postMessage()` [API](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).
```
Comment (ulidtko) [inline source/includes/_iframe.md:41]: s/achieve/achieved/

File: source/includes/_iframe.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+const postData = async (url = '', data = {}) => {
+	try {
+		const response = await fetch(url, {
+			method: 'POST', // *GET, POST, PUT, DELETE, etc.
+			mode: 'cors', // no-cors, *cors, same-origin
+			cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
+			credentials: 'same-origin', // include, *same-origin, omit
+			headers: {
+				'Content-Type': 'application/json'
+				// 'Content-Type': 'application/x-www-form-urlencoded',
```
Comment (ulidtko) [inline source/includes/_iframe.md:90]: `x-www-form-urlencoded` won't ever be useful here, remove this comment?..

**Review by ulidtko — APPROVED** [review]
LGTM with minor suggestions

File: source/includes/_iframe.md (hunk 2 above)
Comment (yowlu) [reply source/includes/_iframe.md:90]: Well, those were some comments to this postData function and explain how any of those parameters can be changed not for our requests to ON/RAMP but in general if somebody wants to reuse it.
Conceptually we could remove all comments here, but I thought it was good for any dev.

File: source/includes/_iframe.md (hunk 1 above)
Comment (yowlu) [reply source/includes/_iframe.md:41]: Can you explain this a little bit more?

**Pull request merged**